diff -wpruN '--exclude=*.orig' a~/src/java.base/share/conf/security/java.security a/src/java.base/share/conf/security/java.security
--- a~/src/java.base/share/conf/security/java.security	1970-01-01 00:00:00
+++ a/src/java.base/share/conf/security/java.security	1970-01-01 00:00:00
@@ -751,7 +751,7 @@
 # Example:
 #   jdk.tls.disabledAlgorithms=MD5, SSLv3, DSA, RSA keySize < 2048, \
 #       rsa_pkcs1_sha1, secp224r1
-jdk.tls.disabledAlgorithms=SSLv3, TLSv1, TLSv1.1, RC4, DES, MD5withRSA, \
+jdk.tls.disabledAlgorithms=SSLv3, RC4, DES, MD5withRSA, \
     DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
     include jdk.disabled.namedCurves
 
