diff -wpruN '--exclude=*.orig' a~/jdk/src/share/lib/security/java.security-solaris a/jdk/src/share/lib/security/java.security-solaris
--- a~/jdk/src/share/lib/security/java.security-solaris	1970-01-01 00:00:00
+++ a/jdk/src/share/lib/security/java.security-solaris	1970-01-01 00:00:00
@@ -701,7 +701,7 @@ jdk.jar.disabledAlgorithms=MD2, MD5, RSA
 #
 # Example:
 #   jdk.tls.disabledAlgorithms=MD5, SSLv3, DSA, RSA keySize < 2048
-jdk.tls.disabledAlgorithms=SSLv3, TLSv1, TLSv1.1, RC4, DES, MD5withRSA, \
+jdk.tls.disabledAlgorithms=SSLv3, RC4, DES, MD5withRSA, \
     DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
     include jdk.disabled.namedCurves
 
